### PR TITLE
draft section by  edoud1

### PR DESCRIPTION
**Pull request by jessegmeyerlab**
moving from jgmeyerucsd repo fork

Please provide a description of your changes:

Please check the following about your suggested changed:
- [x] The new text has one sentence per line
- [x] preprints that are cited are listed as an issue for discussion and peer review

Changes suggested by  @edoud1 - need to review

### CODE REVIEW

File: content/04.protein-extraction.md [hunk 1]
```diff
@@ -1,14 +1,80 @@
...
-4. chemicals to avoid: PEGs, detergents etc
-5. removal of contaminations, Protein Precipitation
-* detergent reomoval resins, S-TRAP (Protifi) etc
-7. protein alkylation
-* choices of reduction and alkylation reagents, TCEP/DTT/2BME, Chloroacetamide/iodoacetamide, n-ethyl maleimide
+Protein extraction from the sample of interest is the initial phase of any mass spectrometry-based proteomics experiment. 
+Thought should be given to any planned downstream assays, specific needs of proteolysis (LiP-MS, post translational modification enrichments, enzymatic reactions, glycan purification or hydrogen-deuterium exchange experiments) long term project goals (reproducibility, multiple sample types, low abundance samples) as well as to the initial experimental question (coverage of a specific protein, subcellular proteomics, global proteomics, protein-protein interactions or immune or affinity enrichment of a specific classes of modifications.) 
+The 2009 version of Methods in Enzymology: guide to Protein Purification, edited by Burgess and Deutscher may serve as a deep dive into how molecular biologists and biochemists have traditionally thought about protein extraction.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]: please add citation
```suggestion
The 2009 version of Methods in Enzymology: guide to Protein Purification [@isbn:978-0-12-374536-1] serves as a deep dive into how molecular biologists and biochemists traditionally thought about protein extraction. 
```

File: content/04.protein-extraction.md [hunk 2]
```diff
@@ -1,14 +1,80 @@
...
+Thought should be given to any planned downstream assays, specific needs of proteolysis (LiP-MS, post translational modification enrichments, enzymatic reactions, glycan purification or hydrogen-deuterium exchange experiments) long term project goals (reproducibility, multiple sample types, low abundance samples) as well as to the initial experimental question (coverage of a specific protein, subcellular proteomics, global proteomics, protein-protein interactions or immune or affinity enrichment of a specific classes of modifications.) 
+The 2009 version of Methods in Enzymology: guide to Protein Purification, edited by Burgess and Deutscher may serve as a deep dive into how molecular biologists and biochemists have traditionally thought about protein extraction. 
+Any change in extraction conditions should be expected to create potential changes in downstream results. 
+Optimize this step first and stick with a protocol that works for your needs. 
+If a collaborator is attempting to reproduce your results/data, make sure they begin with the same extraction protocols.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
If a collaborator is attempting to reproduce your results, make sure they begin with the same extraction protocols. 
```

File: content/04.protein-extraction.md [hunk 3]
```diff
@@ -1,14 +1,80 @@
...
+The 2009 version of Methods in Enzymology: guide to Protein Purification, edited by Burgess and Deutscher may serve as a deep dive into how molecular biologists and biochemists have traditionally thought about protein extraction. 
+Any change in extraction conditions should be expected to create potential changes in downstream results. 
+Optimize this step first and stick with a protocol that works for your needs. 
+If a collaborator is attempting to reproduce your results/data, make sure they begin with the same extraction protocols. 
 
+### Buffer choice 
+#### General proteomics 
+
+A common question to proteomics core facilities is, “What is the best buffer for protein extraction?” – unfortunately, there is no one correct answer.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
A common question to proteomics core facilities is, “What is the best buffer for protein extraction?”
Unfortunately, there is no one correct answer. 
```

File: content/04.protein-extraction.md [hunk 4]
```diff
@@ -1,14 +1,80 @@
...
+The 2009 version of Methods in Enzymology: guide to Protein Purification, edited by Burgess and Deutscher may serve as a deep dive into how molecular biologists and biochemists have traditionally thought about protein extraction. 
+Any change in extraction conditions should be expected to create potential changes in downstream results. 
+Optimize this step first and stick with a protocol that works for your needs. 
+If a collaborator is attempting to reproduce your results/data, make sure they begin with the same extraction protocols. 
 
+### Buffer choice 
+#### General proteomics 
+
+A common question to proteomics core facilities is, “What is the best buffer for protein extraction?” – unfortunately, there is no one correct answer. 
+Typically for a global proteomics experiment a buffer of neutral pH (50-100 mM PBS, Tris, HEPES, Ammonium Bicarbonate; pH 7.5-8.5) is used in conjunction with some chaotropic agent (8 M urea, 6 M guanidine, 5% SDS) [@PMID: 16152629].
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
For global proteomics experiments, a buffer of neutral pH (50-100 mM PBS, Tris, HEPES, Ammonium Bicarbonate, triethanolamine bicarbonate; pH 7.5-8.5) is used in conjunction with a chaotrope or surfactant to denature and solubilize proteins (e.g., 8 M urea, 6 M guanidine, 5% SDS) [@PMID:16152629; @PMID:20722421]. 
Often other salts like 50-150 mM NaCl are also added. 
```

File: content/04.protein-extraction.md [hunk 5]
```diff
@@ -1,14 +1,80 @@
...
+Any change in extraction conditions should be expected to create potential changes in downstream results. 
+Optimize this step first and stick with a protocol that works for your needs. 
+If a collaborator is attempting to reproduce your results/data, make sure they begin with the same extraction protocols. 
 
+### Buffer choice 
+#### General proteomics 
+
+A common question to proteomics core facilities is, “What is the best buffer for protein extraction?” – unfortunately, there is no one correct answer. 
+Typically for a global proteomics experiment a buffer of neutral pH (50-100 mM PBS, Tris, HEPES, Ammonium Bicarbonate; pH 7.5-8.5) is used in conjunction with some chaotropic agent (8 M urea, 6 M guanidine, 5% SDS) [@PMID: 16152629]. 
+If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods – often SDS.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods, such as SDS. 
```

File: content/04.protein-extraction.md [hunk 6]
```diff
@@ -1,14 +1,80 @@
...
+Optimize this step first and stick with a protocol that works for your needs. 
+If a collaborator is attempting to reproduce your results/data, make sure they begin with the same extraction protocols. 
 
+### Buffer choice 
+#### General proteomics 
+
+A common question to proteomics core facilities is, “What is the best buffer for protein extraction?” – unfortunately, there is no one correct answer. 
+Typically for a global proteomics experiment a buffer of neutral pH (50-100 mM PBS, Tris, HEPES, Ammonium Bicarbonate; pH 7.5-8.5) is used in conjunction with some chaotropic agent (8 M urea, 6 M guanidine, 5% SDS) [@PMID: 16152629]. 
+If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods – often SDS. 
+Otherwise, denaturing conditions can efficiently extract a broad range of proteins.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
```

File: content/04.protein-extraction.md [hunk 7]
```diff
@@ -1,14 +1,80 @@
...
 
+### Buffer choice 
+#### General proteomics 
+
+A common question to proteomics core facilities is, “What is the best buffer for protein extraction?” – unfortunately, there is no one correct answer. 
+Typically for a global proteomics experiment a buffer of neutral pH (50-100 mM PBS, Tris, HEPES, Ammonium Bicarbonate; pH 7.5-8.5) is used in conjunction with some chaotropic agent (8 M urea, 6 M guanidine, 5% SDS) [@PMID: 16152629]. 
+If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods – often SDS. 
+Otherwise, denaturing conditions can efficiently extract a broad range of proteins. 
+Compatibility with protease (typically trypsin) and peptide cleanup steps will need to be considered. 
+8 M urea can be diluted to 2 M or less for trypsin and chymotrypsin digestions while guanidine and SDS can be diluted but are best removed either through precipitation of the extracted proteins and resuspension in digestion buffer (see manufacturer’s recommendations) or use of FASP or similar solid phase digestion techniques.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
8 M urea must be diluted to 2 M or less for trypsin and chymotrypsin digestions, while guanidine and SDS should be removed either through protein precipitation, through filter-assisted sample preparation (FASP), or similar solid phase digestion techniques. 
```

Comment (jessegmeyerlab) [reply content/04.protein-extraction.md]: @edoud1 can you please add citations that give more info about protein precipitation, FASP, and and example of solid phase digestion like s-trap and SP3?

File: content/04.protein-extraction.md [hunk 8]
```diff
@@ -1,14 +1,80 @@
...
+If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods – often SDS. 
+Otherwise, denaturing conditions can efficiently extract a broad range of proteins. 
+Compatibility with protease (typically trypsin) and peptide cleanup steps will need to be considered. 
+8 M urea can be diluted to 2 M or less for trypsin and chymotrypsin digestions while guanidine and SDS can be diluted but are best removed either through precipitation of the extracted proteins and resuspension in digestion buffer (see manufacturer’s recommendations) or use of FASP or similar solid phase digestion techniques. 
+Note that some buffers can potentially introduce modifications onto proteins such as carbamylation from urea at high temperatures {@PMID:24161613}. 
+Newer mass spectrometry compatible detergents are also useful for protein extraction and ease of downstream processing – including Rapigest® (Waters), N-octyl-β-glucopyranoside, Azo (Ge lab Wisconsin), PPS silent surfactant.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md:21]: please add citations for these

Comment (jessegmeyerlab) [reply content/04.protein-extraction.md:21]: can we also add mention of other surfactants like sodium laurate, sodium deoxycholate?  for example:
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0059779
https://pubs.acs.org/doi/pdf/10.1021/pr060112a

File: content/04.protein-extraction.md [hunk 9]
```diff
@@ -1,14 +1,80 @@
...
+If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods – often SDS. 
+Otherwise, denaturing conditions can efficiently extract a broad range of proteins. 
+Compatibility with protease (typically trypsin) and peptide cleanup steps will need to be considered. 
+8 M urea can be diluted to 2 M or less for trypsin and chymotrypsin digestions while guanidine and SDS can be diluted but are best removed either through precipitation of the extracted proteins and resuspension in digestion buffer (see manufacturer’s recommendations) or use of FASP or similar solid phase digestion techniques. 
+Note that some buffers can potentially introduce modifications onto proteins such as carbamylation from urea at high temperatures {@PMID:24161613}. 
+Newer mass spectrometry compatible detergents are also useful for protein extraction and ease of downstream processing – including Rapigest® (Waters), N-octyl-β-glucopyranoside, Azo (Ge lab Wisconsin), PPS silent surfactant. 
+AVOID the use of tween-20, triton-X, NP-40, and PEGs as these compounds are challenging to remove after digestion. 
+
+#### Protein-protein interactions
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md:24]: can we cite any resources that would give people direction on good IP buffers in this section?

File: content/04.protein-extraction.md [hunk 10]
```diff
@@ -1,14 +1,80 @@
...
+
+#### Optional additives 
+For denaturing buffer conditions, additional additives may not be necessary for successful extraction and to prevent proteolysis or PTM modifications throughout the extraction process. 
+Protease, phosphatase and deubiquitinase inhibitors are optional additives in less denaturing conditions or in experiments focused on specific post-translational modifications. 
+Keep in mind that protease inhibitors may impact digestion conditions and will need to be diluted or removed prior to trypsin addition. 
+For extraction of DNA or RNA binding proteins, addition of a small amount of benzonase might be useful for degradation of any bound nucleic acids and result in a more consistent digestion.
+
+### Mechanical or Sonic Disruption 
+#### Cell lysis 
+Our laboratories typical lysis buffer is 8 M urea in 100 mM Tris pH 8.5.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
One typical lysis buffer is 8 M urea in 100 mM Tris pH 8.5. 
```

File: content/04.protein-extraction.md [hunk 11]
```diff
@@ -1,14 +1,80 @@
...
+
+### Mechanical or Sonic Disruption 
+#### Cell lysis 
+Our laboratories typical lysis buffer is 8 M urea in 100 mM Tris pH 8.5. 
+Small mammalian cell pellets and exosomes will lyse almost instantly upon addition of this sort of denaturing buffer. 
+Efficiency of extraction and degradation of nucleic acids can be improved using various sonication methods: 1) probe sonicator with ice; 2) water bath sonicator with ice or cooling; 3) bioruptor® sonication device 4) Adaptive focused acoustics (AFA®) [@PMID: 21060726]. 
+Key to these additional lysis techniques are to keep the temperature of the sample from rising significantly which can cause proteins to aggregate or degrade. 
+Some cell types may require additional force for effective lysis (see below). 
+For bacteria with cell wells, lysozyme is often added in the lysis buffer. 
+Any added protein will be present in downstream results, however so excessive addition of lysozyme is to be avoided unless tagged protein purification will occur.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
Any added protein will be present in downstream results, however, so excessive addition of lysozyme is to be avoided unless tagged protein purification will occur. 
```

File: content/04.protein-extraction.md [hunk 12]
```diff
@@ -1,14 +1,80 @@
...
+Key to these additional lysis techniques are to keep the temperature of the sample from rising significantly which can cause proteins to aggregate or degrade. 
+Some cell types may require additional force for effective lysis (see below). 
+For bacteria with cell wells, lysozyme is often added in the lysis buffer. 
+Any added protein will be present in downstream results, however so excessive addition of lysozyme is to be avoided unless tagged protein purification will occur. 
+
+#### Tissue/other lysis 
+Although small pieces of soft tissue can often be successfully extracted with the probe and sonication methods described above, larger/harder tissues as well as plants/yeast/fungi are better extracted with some form of additional mechanical force. 
+If proteins are too be extracted from a large amount of sample – soil, feces, or other diffuse samples – one option is to use a dedicated blender and filter the sample after followed by centrifugation.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
If proteins are too be extracted from a large amount of sample, such as soil, feces, or other diffuse samples, one option is to use a dedicated blender and filter the sample after followed by centrifugation. 
```

File: content/04.protein-extraction.md [hunk 13]
```diff
@@ -1,14 +1,80 @@
...
+Key to these additional lysis techniques are to keep the temperature of the sample from rising significantly which can cause proteins to aggregate or degrade. 
+Some cell types may require additional force for effective lysis (see below). 
+For bacteria with cell wells, lysozyme is often added in the lysis buffer. 
+Any added protein will be present in downstream results, however so excessive addition of lysozyme is to be avoided unless tagged protein purification will occur. 
+
+#### Tissue/other lysis 
+Although small pieces of soft tissue can often be successfully extracted with the probe and sonication methods described above, larger/harder tissues as well as plants/yeast/fungi are better extracted with some form of additional mechanical force. 
+If proteins are too be extracted from a large amount of sample – soil, feces, or other diffuse samples – one option is to use a dedicated blender and filter the sample after followed by centrifugation. 
+If samples are tissue or more concise, cryo homogenization of some sort would be recommended.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
If samples are tissue or more concise, cryo-homogenization is recommended. 
```

Comment (jessegmeyerlab) [reply content/04.protein-extraction.md]: not sure what is meant by "or more concise" please clarify

File: content/04.protein-extraction.md [hunk 14]
```diff
@@ -1,14 +1,80 @@
...
+
+#### Tissue/other lysis 
+Although small pieces of soft tissue can often be successfully extracted with the probe and sonication methods described above, larger/harder tissues as well as plants/yeast/fungi are better extracted with some form of additional mechanical force. 
+If proteins are too be extracted from a large amount of sample – soil, feces, or other diffuse samples – one option is to use a dedicated blender and filter the sample after followed by centrifugation. 
+If samples are tissue or more concise, cryo homogenization of some sort would be recommended. 
+The simplest form of this is grinding the sample with liquid nitrogen and a mortar and pestle, but tools such as bead beaters (sample is placed in a tube with appropriately sized beads and shaken rapidly), cryo-mills (chambers where liquid nitrogen is applied around a vessel and large bead or beads), and cryo-fractionators (samples in special bags are frozen in liquid nitrogen and smashed with various degrees of force) are other options [@PMID: 34002278].
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
The simplest form of this is grinding the sample with liquid nitrogen and a mortar and pestle.
Tools such as bead beaters are also used, where the sample is placed in a tube with appropriately sized beads and shaken rapidly. 
Cryo-mills are chambers where liquid nitrogen is applied around a vessel and large bead or beads. 
Cryo-fractionators homogenize samples in special bags that are frozen in liquid nitrogen and smashed with various degrees of force [@PMID: 34002278]. 
```

File: content/04.protein-extraction.md [hunk 15]
```diff
@@ -1,14 +1,80 @@
...
+
+#### Tissue/other lysis 
+Although small pieces of soft tissue can often be successfully extracted with the probe and sonication methods described above, larger/harder tissues as well as plants/yeast/fungi are better extracted with some form of additional mechanical force. 
+If proteins are too be extracted from a large amount of sample – soil, feces, or other diffuse samples – one option is to use a dedicated blender and filter the sample after followed by centrifugation. 
+If samples are tissue or more concise, cryo homogenization of some sort would be recommended. 
+The simplest form of this is grinding the sample with liquid nitrogen and a mortar and pestle, but tools such as bead beaters (sample is placed in a tube with appropriately sized beads and shaken rapidly), cryo-mills (chambers where liquid nitrogen is applied around a vessel and large bead or beads), and cryo-fractionators (samples in special bags are frozen in liquid nitrogen and smashed with various degrees of force) are other options [@PMID: 34002278]. 
+After homogenization, samples can be sonicated by one of the methods above to fragment DNA/increase solubilization of proteins.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
After homogenization, samples can be sonicated by one of the methods above to fragment DNA and increase solubilization of proteins. 
```

File: content/04.protein-extraction.md [hunk 16]
```diff
@@ -1,14 +1,80 @@
...
+After homogenization, samples can be sonicated by one of the methods above to fragment DNA/increase solubilization of proteins. 
+
+### Efficiency of protein extraction 
+Following protein extraction, samples should be centrifuged (10-14,000 g for 10-30 min depending on sample type) to remove debris prior to calculating protein concentration. 
+The amount of insoluble material remaining should be noted throughout an experiment as a large change may indicate protein extraction issues. 
+Protein concentration can be calculated using a number of assays or tools – generally a UV absorption method such as Bradford or BCA is both facile and affordable. 
+A nanodrop UV spectrophotometer may be used. Consistency in this method is important as each method will have inherent bias and error.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
A nanodrop UV spectrophotometer may be used. 
Consistency in this method is important as each method will have inherent bias and error. 
```

File: content/04.protein-extraction.md [hunk 17]
```diff
@@ -1,14 +1,80 @@
...
+The simplest form of this is grinding the sample with liquid nitrogen and a mortar and pestle, but tools such as bead beaters (sample is placed in a tube with appropriately sized beads and shaken rapidly), cryo-mills (chambers where liquid nitrogen is applied around a vessel and large bead or beads), and cryo-fractionators (samples in special bags are frozen in liquid nitrogen and smashed with various degrees of force) are other options [@PMID: 34002278]. 
+After homogenization, samples can be sonicated by one of the methods above to fragment DNA/increase solubilization of proteins. 
+
+### Efficiency of protein extraction 
+Following protein extraction, samples should be centrifuged (10-14,000 g for 10-30 min depending on sample type) to remove debris prior to calculating protein concentration. 
+The amount of insoluble material remaining should be noted throughout an experiment as a large change may indicate protein extraction issues. 
+Protein concentration can be calculated using a number of assays or tools – generally a UV absorption method such as Bradford or BCA is both facile and affordable.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
Protein concentration can be calculated using a number of assays or tools; generally UV absorption methods are facile and affordable, such as Bradford or BCA assays. 
Protein can also be estimated by tryptophan fluorescence, which has the benefit of not consuming sample [@DOI:10.1021/ac504689z].
```

File: content/04.protein-extraction.md [hunk 18]
```diff
@@ -1,14 +1,80 @@
...
+After homogenization, samples can be sonicated by one of the methods above to fragment DNA/increase solubilization of proteins. 
+
+### Efficiency of protein extraction 
+Following protein extraction, samples should be centrifuged (10-14,000 g for 10-30 min depending on sample type) to remove debris prior to calculating protein concentration. 
+The amount of insoluble material remaining should be noted throughout an experiment as a large change may indicate protein extraction issues. 
+Protein concentration can be calculated using a number of assays or tools – generally a UV absorption method such as Bradford or BCA is both facile and affordable. 
+A nanodrop UV spectrophotometer may be used. Consistency in this method is important as each method will have inherent bias and error. 
+Extraction buffer components will need to be compatible with any assay chosen; alternatively, buffer may be removed (see section V.) prior to protein concentration calculation.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
Extraction buffer components will need to be compatible with any assay chosen; alternatively, buffer may be removed (see below) prior to protein concentration calculation.
```

File: content/04.protein-extraction.md [hunk 19]
```diff
@@ -1,14 +1,80 @@
...
+
+### Reduction and alkylation 
+Typically, proteins are reduced and alkylated prior to proteolysis in order to disrupt structures from disulfide bonds.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md]:
```suggestion
Typically, disulfide bonds in proteins are reduced and alkylated prior to proteolysis in order to disrupt structures and simplify peptide analysis. 
```

File: content/04.protein-extraction.md [hunk 20]
```diff
@@ -1,14 +1,80 @@
...
+A nanodrop UV spectrophotometer may be used. Consistency in this method is important as each method will have inherent bias and error. 
+Extraction buffer components will need to be compatible with any assay chosen; alternatively, buffer may be removed (see section V.) prior to protein concentration calculation.
+
+### Reduction and alkylation 
+Typically, proteins are reduced and alkylated prior to proteolysis in order to disrupt structures from disulfide bonds. 
+This allows better access to all residues during proteolysis and removes the crosslinked peptides created by S-S inter peptide linkages. 
+There are a variety of reagent options for these steps. 
+For reduction, the typical agents used are 5-15 mM concentration of TCEP/DTT/2BME. 
+For the following alkylation step, a slightly higher 10-20mM concentration, alkylating agent such as Chloroacetamide/iodoacetamide, n-ethyl maleimide can be used [PMID: 29019370; @PMID: 15351294; @PMID: 28539326]. 
+It is also possible to alklylate free cysteines with one reagent, reduce di-sulfide bonds (or other Cysteine modifications) and alkylate with a different reagent in order to monitor which residues are linked/modified in a protein.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md:72]: can we cite examples of this?

File: content/04.protein-extraction.md [hunk 21]
```diff
@@ -1,14 +1,80 @@
...
+There are a variety of reagent options for these steps. 
+For reduction, the typical agents used are 5-15 mM concentration of TCEP/DTT/2BME. 
+For the following alkylation step, a slightly higher 10-20mM concentration, alkylating agent such as Chloroacetamide/iodoacetamide, n-ethyl maleimide can be used [PMID: 29019370; @PMID: 15351294; @PMID: 28539326]. 
+It is also possible to alklylate free cysteines with one reagent, reduce di-sulfide bonds (or other Cysteine modifications) and alkylate with a different reagent in order to monitor which residues are linked/modified in a protein. 
+Reduction reactions are generally carried out in the dark at room temperature to avoid excessive off target alklation.
+
+### Removal of buffer/interfering small molecules
+If extraction must take place in a buffer which is incompatible for efficient proteolysis (check the guidelines for the protease of choice), then protein cleanup should occur prior to digestion. 
+This is generally performed through precipitation of proteins. 
+The most common types are 1) acetone, 2) trichloroacetic acid (TCA), and 3) methanol/chloroform/water.
```
Comment (jessegmeyerlab) [inline content/04.protein-extraction.md:78]:
```suggestion
The most common types are 1) acetone, 2) trichloroacetic acid (TCA), and 3) methanol/chloroform/water. 
Proteins are generally insoluble in most pure organic solvents, so cold ethanol or methanol are sometimes used. 
```

**Review by jessegmeyerlab — COMMENTED** [review]
@edoud1 overall this is great thank you!  I suggest some edits and ask for clarification or citations in a few places. If you agree with suggestions you can click the accept buttons. To add citations please use the format [@PMID:number] or [@DOI:10.1021/ac504689z].

**Comment by jessegmeyerlab** [comment]
> @edoud1 overall this is great thank you! I suggest some edits and ask for clarification or citations in a few places. If you agree with suggestions you can click the accept buttons. To add citations please use the format [@pmid:number] or [@doi:10.1021/ac504689z].

@edoud1 to work on this branch use this version https://github.com/jessegmeyerlab/proteomics-tutorial/blob/jessegmeyerlab-edoud1/content/04.protein-extraction.md

**Comment by edoud1** [comment]
Can do - these edits look great and I'll pull some new references/address your comments.

**Review by edoud1 — APPROVED** [review]

**Comment by jessegmeyerlab** [comment]
Thanks Emma - let me know if you need help figuring out how to add the changes, and make sure to add your author info to the metadata.yml on this branch

**Comment by edoud1** [comment]
Hey Jesse,  Can we merge these changes and then I'll add in on those comments/requested citations?  Just to keep things clean and avoid missing any changes. It's not giving me the option to 'approve and merge' unless I'm missing it somewhere! Which I fully admit I might be. I'll try to get my end updated before ASMS - see you there?

**Comment by jessegmeyerlab** [comment]
> Hey Jesse, Can we merge these changes and then I'll add in on those comments/requested citations? Just to keep things clean and avoid missing any changes. It's not giving me the option to 'approve and merge' unless I'm missing it somewhere! Which I fully admit I might be. I'll try to get my end updated before ASMS - see you there?

You are correct I am the only one who can approve and merge on the main branch of this repository as of right now. Yes I will be at ASMS and chairing the session on AI in MS research Tuesday afternoon!

Before we merge this, you can accept each of the suggested changes if you agree by clicking the "commit suggestion" buttons:
image

After that, it should be okay to merge this for now and then have you add edits on top of the main branch again in a new pull request (PR). 

Just so you know how to add to this branch for this PR (and in case anyone else reads this who wants to know how), if you wanted to add those edits on this PR, you can open the branch that this PR comes from by clicking on the branch at the top and then add the edits there. 
image

---------------------------------------------------
------------------------------------------------------------
You can check that that file is on the correct branch of the PR 
-------------------
image

-------------------------------------------------------------
and then commit changes making sure they are again to that branch being considered for a PR:
-----------------------------------------
image

**Comment by edoud1** [comment]
I'm not seeing the option to "commit changes" anywhere?
image

**Comment by jessegmeyerlab** [comment]
ok thanks for letting me know. maybe its because I made this pull request. I will accept everything and we can try having you add new changes to the base branch again.

**Comment by jessegmeyerlab** [comment]
OK its done! thanks for your help

**Comment by jessegmeyerlab** [comment]
@edoud1 I noticed that some of the citations are not formatted correctly so they are misbehaving in the HTML version that is automatically generated. Can you please fix these in you new pull request if you notice them?  The format should be [@PMID:34002278] 
image

**Comment by edoud1** [comment]
Thanks for catching - I'll update on the next round where I address your earlier comments!  On the to do list for prior to the plane on Saturday....

**Pull request merged**